### PR TITLE
Fix #1932

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: fuzzers/frida_libpng/src/fuzzer.rs [hunk 1]
```diff
@@ -140,7 +140,7 @@ unsafe fn fuzz(options: &FuzzerOptions) -> Result<(), Error> {
                         // RNG
                         StdRand::with_seed(current_nanos()),
                         // Corpus that will be evolved, we keep it in memory for performance
-                        CachedOnDiskCorpus::no_meta(PathBuf::from("./corpus_discovered"), 64)
+                        CachedOnDiskCorpus::no_meta(PathBuf::from("./corpus_discovered"), 4)
```
Comment (domenukk) [inline fuzzers/frida_libpng/src/fuzzer.rs:143]: For an example fuzzer this might be a bit small?

Comment (tokatoka) [reply fuzzers/frida_libpng/src/fuzzer.rs:143]: yeah that was for debugging
i forgot to remove

File: libafl/src/corpus/cached.rs [hunk 2]
```diff
@@ -125,14 +119,14 @@ where
     #[inline]
     fn get(&self, idx: CorpusId) -> Result<&RefCell<Testcase<I>>, Error> {
         let testcase = { self.inner.get(idx)? };
-        self.cache_testcase(testcase, idx, false)?;
+        self.cache_testcase(testcase, idx)?;
         Ok(testcase)
     }
     /// Get by id; considers both enabled and disabled testcases
     #[inline]
     fn get_from_all(&self, idx: CorpusId) -> Result<&RefCell<Testcase<Self::Input>>, Error> {
```
Comment (domenukk) [inline libafl/src/corpus/cached.rs:127]: Should we even cache if the testcase is disabled? I don't know if we usually want that, it should be used way less?

**Comment by R9295** [comment]
@domenukk Oh, my bad. 

If I understood it correctly, it is because the disabled entry may be considered as enabled when the fuzzer restarts?

**Comment by tokatoka** [comment]
no it's unrelated. see the end of #1932

**Pull request merged**